PublishToDocker.groovy

```diff
@@ -0,0 +1,54 @@
+pipeline {
+  agent any
+  environment {
+    DOCKER_CREDENTIALS = credentials('docker-credentials')
+  }
+  stages {
+    stage('Log into docker registry') {
+      steps {
+        sh "docker login -u $DOCKER_CREDENTIALS_USR -p $DOCKER_CREDENTIALS_PSW"
+      }
+    }
+    stage('Build and publish fixtures generator') {
+      steps {
+        script {
+          if(BUILD_TYPE == 'production') {
+            serviceName = 'fixtures-generator'
+            dockerRepoName = "$DOCKER_CREDENTIALS_USR/$serviceName:${TAG}"
+            sh "docker build -t $dockerRepoName --file ./$serviceName/Dockerfile ."
+            sh "docker push $dockerRepoName"
+          }
+        }
+      }
+    }
+    stage('Build and publish database service') {
+      steps {
+        script {
+          if(BUILD_TYPE == 'production') {
+            serviceName = 'database-service'
+            dockerRepoName = "$DOCKER_CREDENTIALS_USR/$serviceName:${TAG}"
+            sh "docker build -t $dockerRepoName --target app --file ./$serviceName/Dockerfile ."
+            sh "docker push $dockerRepoName"
+          }
```

Comment:
It's not clear yet what should be packed into the e2e backend images. Therefore, only the production image is defined at the moment.